modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/UserAccountResourceImpl.java

```diff
@@ -1555,6 +1544,26 @@ private List<Website> _getWebsites(User user, UserAccount userAccount) {
 			Objects::nonNull);
 	}
 
+	private Integer _getWorkflowStatus(String status) throws Exception {
```

Comment:
Is is possible to use the same method also for the PATCH? Something like ... in this method we are not throwing exception (but returns null in case)
Where we are using it (I think in PUT) we add a if (Validator.isNull() throw new***, in PATCH if (Validator.isNotNull()) updateStatus. 
That's because I don't like to have this "fromString to int" twice in the class (like it is now, not for your changes). Do you have any other suggestion?